src/drone_controller/input_layer/drone_state.py

```diff
@@ -0,0 +1,72 @@
+# pylint: disable=too-many-arguments
+# a wrapper class needs more parameter
+from src.drone_controller.exception.exceptions import DroneControllerError
+
+
+class DroneState:
+    """
+    Wrapper class for the state of the drone.
+    """
+
+    def __init__(self, position, velocity, velocity_ang, acceleration, acceleration_ang):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing docstring in __init__](https://app.codacy.com/gh/treeandsea/DroneController/pullRequest?prid=5205178)